src/util/codec/mysql/json.rs

```diff
@@ -0,0 +1,696 @@
+// Copyright 2017 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use super::Result;
+use std::{str, f64};
+use std::cmp::{Ord, Ordering, PartialEq, PartialOrd};
```

Comment:
`Ord` `PartialEq`, `PartialOrd`, are in prelude. 
you'll see different color in github code view. 😆